packages/documentation/src/stories/foundation/color/color.stories.mdx

```diff
@@ -0,0 +1,119 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import '@swisspost/design-system-styles/basics.scss';
+import './color.styles.scss';
+import { cssBackground, scssBackground } from './color.snippets';
+import { ColorSwatch } from './color-swatch';
+
+<Meta
+  title="Foundations/Color"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+  }}
+/>
+
+# Color
+
+You can find the [color usage documentation](#usage) right below.
+
+## Brand colors
+
+<section className="color-square__section">
+  {['yellow', 'black', 'white'].map(color => (
+    <ColorSwatch key={color} color={color} />
+  ))}
+</section>
+
+## Shades of gray
+
+<section className="color-square__section">
+  {['gray-10', 'gray-20', 'gray-40', 'gray-60', 'gray-80'].map(color => {
+    return (
+      <article key={color}>
+        <div className="color-square">
+          <div
+            className="color-square__inner"
+            style={{ backgroundColor: `var(--post-${color}` }}
+          ></div>
+          <div class="color-square__content">
+            <h3>{color}</h3>
+            <dl>
+              <dt>CSS</dt>
+              <dd>
+                <code>var(--post-{color})</code>
+              </dd>
+              <dt>Sass</dt>
+              <dd>
+                <code>post.${color}</code>
+              </dd>
+            </dl>
+          </div>
+        </div>
+      </article>
+    );
+  })}
+</section>
```

Comment:
Why don't you use the same ColorSwatch Component here as above?
If it's only because of hte "h3", you could make the rendering of it conditionally.

Reply:
Improved reusability of the component